pep-0013.rst

```diff
@@ -3,6 +3,7 @@ Title: Python Language Governance
 Author: The Python core team and community
 Status: Active
 Type: Informational
```

Comment:
```suggestion
Type: Process
```

While we're at it, this should really be fixed...PEP 13 is perhaps the most normative, process and "meta" of all PEPs, describing the fundamental governance model of the Python community, which in turn governs PEP 1 and the PEP process itself (among other things). It doesn't really make sense to categorize this as a mere non-normative whitepaper/recommendation, nor to call PEP 609 (PyPA governance) as Process-level but Python's own governance as "Informational".

Reply:
I should learn what "normative" means, I have to look it up every time you use it! 🙃